programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md

```diff
@@ -18,4 +18,15 @@
 
 ## Proposed Project ideas
 
+### Cilium
+
+#### Governance Documentation
+
+- Description: This project will focus on governance documentation for the Cilium project with two key parts. First, the governenace documentation should be moved out of the main docs and into the community repo. Second, we need to do an inventory of all of the repos under the project and come up with a lifecycle for them.
+- Expected Outcome: Governance docs in community repo. All repos accounted for and with a lifecycle plan.
+- Recommended Skills: enthusiasm for governance and basic markdown experience
+- Mentor(s):
+  - Mentor Name (@mentor_github, mentor@email.addy) - please use the same email address as you use on the LFX Mentorship Platform at https://mentorship.lfx.linuxfoundation.org
```

Comment:
Could you update the mentor information?

Also, could you please add a back up mentor? This year, we really encourage having 2 mentors in all programs.

Reply:
Sorry, totally missed that section 😅